resources/views/layout/navigation.blade.php

```diff
@@ -0,0 +1,88 @@
+            <a class="my-4 p-2 rounded-2xl">{{ __("content.pages.case_study") }}</a>
+        </button>
+        <button class="w-full nav-item p-8 @if($current === "partners") active @endif" onclick="window.location='{{ route("partners") }}';">
+            <a class="my-4 p-2 rounded-2xl">{{ __("content.pages.partners") }}</a>
+        </button>
+        <button class="w-full nav-item p-8 @if($current === "career") active @endif" onclick="window.location='{{ route("career") }}';">
+            <a class="my-4 p-2 rounded-2xl">{{ __("content.pages.career") }}</a>
+        </button>
+        <button class="w-full nav-item p-8 @if($current === "contact") active @endif" onclick="window.location='{{ route("contact") }}';">
+            <a class="my-4 p-2 rounded-2xl">{{ __("content.pages.contact") }}</a>
+        </button>
+        <div class="w-full p-8">
+            @foreach(LocaleConfig::getLocales() as $locale)
+                @if (!App::isLocale($locale))
+                    <a href="{{ route(Route::currentRouteName(), Route::getCurrentRoute()->parameters(), true, $locale) }}">
+                        <i class="large {{ $locale === "en" ? "gb" : $locale }} flat flag"></i>
+                    </a>
+                @endif
+            @endforeach
+        </div>
+    </div>
+    <x-primary-button-with-icon icon="text" href="">{{ __("buttons.lets_talk") }}</x-primary-button-with-icon>
+</nav>
+
+<nav id="desktop-nav" class="left-0 right-0 py-4 duration-300 xl:px-[15%] bg-navigation relative">
+    <div class="mx-auto flex items-center justify-between px-6">
+        <div>
+            <a class="flex items-start" href="{{ route("home") }}">
+                <img class="w-40 shrink-0" src="/logo.svg" alt="Blumilk"/>
+            </a>
+        </div>
+        <div class="md:flex hidden justify-center items-center text-lg lg:text-xl">
+            <p class="font-medium flex items-center text-center gap-2 nav-list">
+                <a onclick="{{route('about')}}" class="nav-item @if($current === "about") active @endif" href="{{ route("about") }}">{{ __("content.pages.about") }}</a>
+                <a href="{{ route("case-studies") }}" class="nav-item @if($current === "case-studies") active @endif">{{ __("content.pages.case_study") }}</a>
+                <a href="{{ route("partners") }}" class="nav-item @if($current === "partners") active @endif">{{ __("content.pages.partners") }}</a>
+                <a href="{{ route("career") }}" class="nav-item @if($current === "career") active @endif">{{ __("content.pages.career") }}</a>
+                <a href="{{ route("contact") }}" class="nav-item @if($current === "contact") active @endif">{{ __("content.pages.contact") }}</a>
+            </p>
+            <div class="m-2">
+                <x-primary-button-with-icon icon="text" href="">{{ __("buttons.lets_talk") }}</x-primary-button-with-icon>
```

Comment:
Why empty `href` on button ?

Reply:
It will be filled with href, is # for now